portfolio/src/main/webapp/paragraph.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Adds a headig with a paragraph to each picture.
+ */
+let files = getImages("/images");
+let descriptions = ["This picture was taken accidentally by my friend two years ago. However it becomes one of my favorite photo since at that time I was still quite slim.",
+                    "Me & Welcome Packge with Google!",
+                    "Welcome package from Google!",
+                    "My favorite ID photo, since I was truly slim at that time."];
+let size = files.length;
+for (let i = 0; i < size; i++) {
+    let tag = document.createElement("a");
```

Comment:
Maybe abstract these html creation out to a separate method?
for (..) {
     let imageContainer = createImageAndDescription(imageName, description)
     append imageContainer
}

Reply:
Fixed